module/LineTraceArea.cpp

```diff
@@ -58,6 +48,6 @@ void LineTraceArea::runLineTraceArea(const bool isLeftCourse, bool& isLeftEdge,
   logger.logHighlight(buf);
   for(int i = 0; i < size; i++) {
     // Linetracer::runに区間の情報を渡して走行させる
-    lineTracer.run(DISTANCE[i] + PARAM[i].tweak, targetBrightness, PARAM[i].pwm, PARAM[i].pidGain);
+    lineTracer.run(params[i].distance, targetBrightness, params[i].pwm, params[i].pidGain);
```

Comment:
ここ区間距離は 計測距離 + 調整距離 をイメージしてたんだけどまとめる？
計測距離は定数としてこのクラスが持ってた方がいいのかなって思ったけどまとめた方がいい？
これはみんなの意見を聞きたいところでもあるけど
もしくは何か理由があってくっつけてる？
```suggestion
    lineTracer.run(DISTANCE[i] + params[i].tweak, targetBrightness, params[i].pwm, params[i].pidGain);
```

Reply:
クラスで持つと、区間数を変えたい場合にクラス内の定数を変える必要が出てくるので、どちらも外部ファイルでできた方がいいと思いました。
比較的みづらいcsvファイルで区間の長さと調整距離をファイル等に分けるのもどうかと思いまとめました。